tox.ini

```diff
@@ -3,7 +3,7 @@ envlist = py34, py35, flake8, rstcheck
 
 [testenv]
 deps = pytest
-commands = py.test
+commands = py.test -s
```

Comment:
The `-s` argument got introduced so that `tox` is not reporting the following errors:

```
conductr_cli/test/test_bndl_create.py:371: AssertionError
------------------------------------------------------------------- Captured stderr call -------------------------------------------------------------------
Error: bndl: redirected Stdin is pseudofile, has no fileno()
```

The "error" was introduced by this line:

```
has_stdin = not os.isatty(sys.stdin.fileno())
```

More info: https://stackoverflow.com/questions/25182812/using-python-pudb-debugger-with-pytest